ts/Core/Pointer.ts

```diff
@@ -1965,9 +1965,14 @@ class Pointer {
         // Update positions (regardless of kdpoint or hoverPoint)
         } else if (followPointer && tooltip && !tooltip.isHidden) {
             anchor = tooltip.getAnchor([{} as any], e);
-            tooltip.updatePosition(
-                { plotX: anchor[0], plotY: anchor[1] } as any
-            );
+            if (
```

Comment:
It looks that we are not using the `anchor` variable in any other place than the one inside of visible if block so we should be able to simply move the second if inside the first one:

```
followPointer &&
tooltip && !tooltip.isHidden &&
tooltip.isInsideX(anchor[0]) &&
tooltip.isInsideY(anchor[1])
```

Reply:
But then you need to move the `getAnchor` outside without really gaining a whole lot.

Reply:
I am not sure if we need to move it outside - I cannot find any other place where it is used

Reply:
Yes, that is right. So we can leave it as it is for now :)